src/figuresClasses.ts

```diff
@@ -1,19 +1,94 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape : Shape,
+  color : Color,
+  getArea() :number,
```

Comment:
proper formatting, fix it everywhere
```suggestion
  shape: Shape,
  color: Color,
  getArea(): number,
```